### DISCUSSION TITLE
Retrocompatibility for login, log in, logout, log out

### DISCUSSION THREAD

**Discussion by caouecs** [Laravel-Lang/common#51] [Q&A]
Hello everybody,
sorry I don't know to add a poll :angel: 

But are you ok to delete tags with login and logout ?

Or, you prefer to have login, logout and log in log out, for retrocompatibility ?

**Comment by luisprmat** [comment]
I agree to keep the labels with 'login' and 'logout' during version 8 (mark them as deprecated), but delete them from version 9

**Reply by masterix21** [reply]
I think it's better to keep both versions.

**Reply by andrey-helldar** [reply]
Keeping extra keys is storing garbage.
I agree with @luisprmat that now they should be left as deprecated, and in the 9th version they should be removed unused in Laravel products.
